bin/breakglass-db-access

```diff
@@ -0,0 +1,118 @@
+#! /bin/bash
+
```

Comment:
Consider `set -e` so that errors are easy to detect and stop the script from running.  Also perhaps `set -x` so that we can find out what's running.

Reply:
Added `set -e`. I printed most relevant information (except DB password) so `set -x` may not be necessary. It can always be added to debug.